graph/impl/src/main/java/org/jboss/windup/graph/GraphContextImpl.java

```diff
@@ -85,6 +87,7 @@ public GraphContextImpl create()
         FileUtils.deleteQuietly(graphDir.toFile());
         TitanGraph titan = initializeTitanGraph();
         initializeTitanIndexes(titan);
+        generateTypeScriptModels();
```

Comment:
why are generated models everytime GraphContext is created? Aren't they the same until you change the code for Model (compile time) ?